### PR TITLE
query simplified

### PR DESCRIPTION
**Pull request by richagaur**

### CODE REVIEW

File: AzureCosmosDBForNoSQL/cosmos_recommendation.ipynb [hunk 1]
```diff
@@ -277,15 +254,9 @@
         "    results = product_catalog_container.query_items(\n",
         "        query='''\n",
         "            SELECT TOP @num_results \n",
-        "                c.id, c.Id, c.Type, c.Brand, c.Name, c.Description, c.Price, c.SimilarityScore\n",
-        "            FROM \n",
-        "                (SELECT \n",
-        "                    s.id, s.Id, s.Type, s.Brand, s.Name, s.Description, s.Price, \n",
-        "                    VectorDistance(s.Embedding, @embedding) AS SimilarityScore\n",
-        "                FROM s \n",
-        "                WHERE ARRAY_CONTAINS(@product_ids, s.id)) \n",
```
Comment (markjbrown) [inline AzureCosmosDBForNoSQL/cosmos_recommendation.ipynb:286]: @richagaur why did you remove the product ids from here?  Did you notice the similarity score is much lower in the results below?

Comment (richagaur) [reply AzureCosmosDBForNoSQL/cosmos_recommendation.ipynb:286]: @markjbrown the productIds are there in the query

query='''\n",
        "            SELECT TOP @num_results \n",
        "                c.id, c.Id, c.Type, c.Brand, c.Name, c.Description, c.Price, VectorDistance(c.Embedding, @embedding) AS SimilarityScore\n",
        "            FROM c WHERE **ARRAY_CONTAINS(@product_ids, c.id)** \n",
        "            ORDER BY c.SimilarityScore DESC''',\n",

Comment (markjbrown) [reply AzureCosmosDBForNoSQL/cosmos_recommendation.ipynb:286]: strange, somehow it looked like that line was removed. ok, all good.
